covasim/webapp/cova_app.js

```diff
@@ -334,7 +334,7 @@ var vm = new Vue({
             this.intervention_figs = {};
             this.setupFormWatcher('sim_pars');
             this.setupFormWatcher('epi_pars');
-            this.result.graphs = [];
+            // this.result.graphs = [];
```

Comment:
Better to not clear the graphs to make it easier to compare results between runs

Reply:
I'm not sure how valuable that is if history is no longer available, especially since this is the reset function.

Reply:
It's valuable. :)